test/fixtures/single-account-ecs-with-autoscaling/main.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
+  required_providers {
+    sysdig = {
+      source  = "sysdiglabs/sysdig"
+      version = ">=0.5.33"
+    }
+  }
+}
+
+provider "sysdig" {
+  sysdig_secure_api_token = var.sysdig_secure_api_token
+  sysdig_secure_url       = var.sysdig_secure_url
+}
+
+provider "aws" {
+  region = var.region
+}
+
+module "cloudvision_aws_single_account_ecs_with_autoscaling" {
+  source = "../../../examples/single-account-ecs"
+  name   = "${var.name}-single"
+
```

Comment:
maybe re-use existing test `/test/fixtures/single-account-ecs` to lower maintenance?
that way it will be tested by GH action too, even if it will not wait for autoscale to kick in

Reply:
I wanted to make an example with autoscaling, to check if the previous one was being deployed correctly, but since the feedback loop will be longer, I will just reuse the existing one, you are right.